tests/test_snyk_reporting.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+
+from komodo.snyk_reporting import snyk_main
+from komodo.symlink.suggester.release import Release
+
+from unittest.mock import patch, Mock, PropertyMock
+import sys
+
+if sys.version_info >= (3, 7):
+    from snyk.models import Vulnerability
+    from snyk import SnykClient
+
+above_py37 = pytest.mark.skipif(
+    sys.version_info < (3, 7), reason="requires Python >= 3.7"
+)
+
+
+def _create_result_mock(issue_ids):
+
+    result_mock = Mock()
+    result_mock.issues.vulnerabilities = [
+        Vulnerability(
+            id=issue_id,
+            url="some_url",
+            title="some_title",
+            description="some_description",
+            upgradePath="some_upgradePath",
+            package="some_package",
+            version="some_version",
+            severity="some_severity",
+            exploitMaturity="some_exploitMaturity",
+            isUpgradable="some_isUpgradable",
+            isPatchable="some_isPatchable",
+            isPinnable="some_isPinnable",
+            identifiers="some_identifiers",
+            semver="some_semver",
+        )
+        for issue_id in issue_ids
+    ]
+    return result_mock
+
+
+@above_py37
+def test_no_api_token():
+    with pytest.raises(
+        ValueError,
+        match=r"No api token given, please set the environment variable SNYK_API_TOKEN.",
+    ):
+        vulnerabilities = snyk_main(
```

Comment:
vulnerabilities is unused

Reply:
I know, but I left it as this because you could easily see the interface from the tests 🙂

Reply:
Makes sense